aws/csi-driver/main.tf

```diff
@@ -0,0 +1,145 @@
+data "aws_eks_cluster" "cluster" {
```

Comment:
Consider adding README for each module, or at least some basic explanation on what's the purpose of each sub-module here

Reply:
I think for now it redundant